### DISCUSSION TITLE
Is this supposed to come with starship?

### DISCUSSION THREAD

**Discussion by rv178** [starship/starship#3007] [Q&A] [answered]
# Is there any way of changing these?

I had installed starship yesterday for my zsh shell (along with zsh-syntax-highlighting and zsh-autosuggestions) and then I found these between the prompt's presets. (the `in`, `is` and `via`)

Any help would be highly appreciated. 

![image](https://user-images.githubusercontent.com/69449141/130902653-9aa8de5f-c3f6-4208-be2b-993484a3240a.png)

**Comment by vladimyr** [comment, marked as answer]
Each of those is part of the corresponding module's format string which is configurable, of course. Take a look at the node for example https://starship.rs/config/#options-39

**Reply by rv178** [reply]
> Each of those is part of the corresponding module's format string which is configurable, of course. Take a look at the node for example https://starship.rs/config/#options-39

Tysm!
